extensions/raycast-gemini/src/aiChat.jsx

```diff
@@ -222,6 +222,20 @@ export default function Chat({ launchContext }) {
             }}
             shortcut={{ modifiers: ["cmd", "shift"], key: "arrowUp" }}
           />
+          <Action
+            icon={Icon.Clipboard}
+            title="Append Selected Text"
+            onAction={async () => {
+              try {
+                const selectedText = await getSelectedText();
+                setSearchText((oldText) => oldText + selectedText);
+              } catch (error) {
+                console.error(error);
```

Comment:
Same as https://github.com/raycast/extensions/pull/16785#discussion_r1939012519. Please don't log any useless errors to the console.